types/dialogflow/index.d.ts

```diff
@@ -861,9 +861,15 @@ export interface Intent {
     followupIntentInfo?: FollowupIntentInfo[];
 }
 
+export enum PhraseType {
```

Comment:
In v2 this is known as a `Type` inside of `TrainingPhrase` and it is a number type
See https://github.com/googleapis/nodejs-dialogflow/blob/d8c259ec1e24cf8412e8d6a79ac1e08ef11e57c5/src/v2/doc/google/cloud/dialogflow/v2/doc_intent.js#L188